plasmapy/particles/particle_class.py

```diff
@@ -118,6 +118,12 @@ def __bool__(self):
         """
         raise AtomicError("The truth value of a particle is not defined.")
 
+    def to_json(self, dict_representation) -> str:
+        """
+        Return a JSON string representation of the minimum description of a particle.
+        """
+        return json.dumps(dict_representation)
```

Comment:
I think there's some rule in programming that I'm forgetting the name of that says if a subclass implements the same method as a parent class, it should have the same calling signature. I think it'd be nice to make this have no arguments and call `self.to_dict()`, which could `raise NotImplementedError` in the base class. That could also be an `@abstractmethod`, so that if you subclass it, you have to remember to implement `to_dict`.

Reply:
Thank you for the pointer. I'll make the necessary changes for it.